readme.md

```
@@ -1,134 +1,324 @@
+- **IdP** — Keycloak (OIDC) for real SSO; per-role demo login for evaluation. ✅
+- **AI service** — RAG over the collective agreement (**Voyage `voyage-3.5` embeddings + pgvector + Anthropic Claude**, answers always cite article + BOCM page, similarity threshold against hallucination) and **Text-to-SQL** with a 12-table whitelist without PII, deterministic SQL guardrails and `READ ONLY` execution. ✅ Final.
+- **Synchronization worker** — direct read of the BioStar and SAGE SQL DBs. ⏸ Descoped (client credentials not granted); architecture specified, post-delivery work.
 
-> Describe los componentes más importantes, incluyendo la tecnología utilizada
+### **2.3. High-level project description and file structure**
 
-### **2.3. Descripción de alto nivel del proyecto y estructura de ficheros**
+Monorepo with modular organization by domain:
 
-> Representa la estructura del proyecto y explica brevemente el propósito de las carpetas principales, así como si obedece a algún patrón o arquitectura específica.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)